extensions/2.0/Vendor/MSFT_audio_emitter/README.md

```diff
@@ -0,0 +1,356 @@
+|**innerAngle**|`number`|Size of cone in radians for a directional sound in which there will be no attenuation.|No, default: `3.141592653589793`|
+|**outerAngle**|`number`|Size of cone in radians for a directional sound outside of which there will be no sound. Listener angles between innerAngle and outerAngle will falloff linearly.|No, default: `3.141592653589793`|
+|**loop**|`boolean`|Selected source must be played in loops.|No, default: `false`|
+|**volume**|`number`|Playback volume.|No, default: `1`|
+|**clips**|[`Emitter Clip`](#reference-emitter-clip) `[1-*]`|| :white_check_mark: Yes|
+|**name**|`string`|The user-defined name of this object.|No|
+|**extensions**|`object`|Dictionary object with extension-specific objects.|No|
+|**extras**|`any`|Application-specific data.|No|
+
+#### emitter.distanceModel
+
+Attenuation function to use on the audio source as it moves away from the listener. The attenuation calculation is done in emitter space.
+> Example: Emitter refDistance is set to 1 and the emitter's node is scaled by a factor of 10 the refDistance will express as if it was set to 10, rather than unchanged at 1.
+
+* **Type**: `string`
+* **Required**: No, default: `exponential`
+* **Allowed values**:
+  * `linear` expressed by: `1 - rolloffFactor * (distance - refDistance) / (maxDistance - refDistance)`
+  * `inverse` expressed by: `refDistance / (refDistance + rolloffFactor * (distance - refDistance))`
+  * `exponential` expressed by: `pow(distance / refDistance, -rolloffFactor)`
+
+#### emitter.refDistance
+
+A reference distance for reducing volume as source moves further from the listener.
+
+* **Type**: `number`
+* **Required**: No, default: `1`
+* **Minimum**: `> 0`
+
+### emitter.maxDistance
+
+The maximum distance between source and listener outside of which there will be no sound. Must be larger than refDistance and only applies to linear distanceModel.
+
+* **Type**: `number`
+* **Required**: No, default: `256`
+* **Minimum**: `> 0`
+
+#### emitter.rolloffFactor
+
+Factor at which attenuation occurs as the source moves further from the listener. Value must be <= 1 if distanceModel is linear, unbounded otherwise.
+
+* **Type**: `number`
+* **Required**: No, default: `1`
+* **Minimum**: `>= 0`
+
+#### emitter.innerAngle
+
+The cone will be symmetrical and face the forward direction (Z+). The outerAngle value must be greater than or equal to the innerAngle value. If neither innerAngle or outerAngle are defined the emitter will behave as an omnidirectional source.
+
+* **Type**: `number`
+* **Required**: No, default: `3.141592653589793`
+* **Minimum**: `>= 0`
+* **Maximum**: `<= 3.141592653589793`
+
+#### emitter.outerAngle
+
+The cone will be symmetrical and face the forward direction (Z+). The outerAngle value must be greater than or equal to the innerAngle value. If neither innerAngle or outerAngle are defined the emitter will behave as an omnidirectional source.
```

Comment:
This is inconsistent with KHR_lights_punctual which defines spotlights as facing -Z. Is it expected that conical audio should be pointing the opposite direction as conical lights?